ports/sdl3/portfile.cmake

```
@@ -28,6 +28,10 @@ endif()
 if ("ibus" IN_LIST FEATURES)
     message(WARNING "You will need to install ibus dependencies to use feature ibus:\nsudo apt install libibus-1.0-dev\n")
 endif()
+# option for not need to show windows
+if (VCPKG_CMAKE_SYSTEM_NAME STREQUAL "Linux" AND NOT "x11" IN_LIST FEATURES AND NOT "wayland" IN_LIST FEATURES)
+    list(APPEND FEATURE_OPTIONS -DSDL_UNIX_CONSOLE_BUILD=ON)
+endif()
```

It looks like the only use of this option is to suppress emitting this error because that is probably not what the user meant: https://github.com/libsdl-org/SDL/blob/f2bcfe3dd2a5d630751369be3e3e88f8d83acd92/cmake/macros.cmake#L378

Moreover it seems that this is intending to talk about the target machine.

How do you feel about something like this instead?

```suggestion
list(APPEND FEATURE_OPTIONS -DSDL_UNIX_CONSOLE_BUILD=ON)
if (VCPKG_TARGET_IS_LINUX AND NOT "x11" IN_LIST FEATURES AND NOT "wayland" IN_LIST FEATURES)
    message(WARNING "The selected features don't allow sdl3 to create windows, which is usually unintentional. You can get windowing support by installing the x11 and/or wayland features.")
endif()
```

---

@BillyONeal 
I agree with you!
Fixed.